chia/_tests/wallet/vault/test_vault_wallet.py

```diff
@@ -185,16 +185,6 @@ async def test_vault_creation(
     assert vault_info == wallet.vault_info
     assert vault_info.recovery_info == wallet.vault_info.recovery_info
 
-    # test make_solution
-    coin = (await wallet.select_coins(uint64(100), DEFAULT_COIN_SELECTION_CONFIG)).pop()
-    wallet.make_solution(primaries, coin_id=coin.name())
-    with pytest.raises(ValueError):
-        wallet.make_solution(primaries)
-
-    # test match_hinted_coin
-    matched = await wallet.match_hinted_coin(wallet.vault_info.coin, wallet.vault_info.inner_puzzle_hash)
-    assert matched
-
```

Comment:
Are these changes intentional?

Reply:
yeah, those lines were just for coverage and won't be needed once the make_solution changes are done

Reply:
Oh I see, I thought these were not vault tests, my bad.